package test;

public class ExceptionEx1_success {
	public static void main(String[] args) {
		Integer num = 100;
		System.out.println(num);
	}
}
